packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -666,34 +678,33 @@ export class GarbageCollector implements IGarbageCollector {
 	 * 2. Clears tracking for deleted nodes.
 	 *
 	 * @param gcResult - The result of the GC run on the gcData.
+	 * @param tombstoneReadyNodes - List of nodes that are tombstone ready.
 	 * @param sweepReadyNodes - List of nodes that are sweep ready.
-	 * @param currentReferenceTimestampMs - The timestamp to be used for unreferenced nodes' timestamp.
-	 * @param logger - The logger to be used to log any telemetry.
 	 * @returns A list of nodes that have been deleted.
 	 */
-	private runSweepPhase(gcResult: IGCResult, sweepReadyNodes: Set<string>): string[] {
+	private runSweepPhase(
+		gcResult: IGCResult,
+		tombstoneReadyNodes: Set<string>,
+		sweepReadyNodes: Set<string>,
+	): string[] {
 		/**
-		 * Currently, there are 3 modes for sweep:
-		 * Test mode - Unreferenced nodes are immediately deleted without waiting for them to be sweep ready.
-		 * Tombstone mode - Sweep ready modes are marked as tombstones instead of being deleted.
-		 * Sweep mode - Sweep ready modes are deleted.
+		 * Under "Test Mode", Unreferenced nodes are immediately deleted without waiting for them to be sweep ready.
 		 *
-		 * These modes serve as staging for applications that want to enable sweep by providing an incremental
-		 * way to test and validate sweep works as expected.
+		 * Otherwise, depending on how long it's been since the node was unreferenced, it will either be
+		 * marked as Tombstone, or deleted by Sweep.
 		 */
 		if (this.configs.testMode) {
 			// If we are running in GC test mode, unreferenced nodes (gcResult.deletedNodeIds) are deleted.
 			this.runtime.updateUnusedRoutes(gcResult.deletedNodeIds);
 			return [];
 		}
 
-		if (this.configs.tombstoneMode) {
-			this.tombstones = Array.from(sweepReadyNodes);
-			// If we are running in GC tombstone mode, update tombstoned routes. This enables testing scenarios
-			// involving access to "deleted" data without actually deleting the data from summaries.
-			this.runtime.updateTombstonedRoutes(this.tombstones);
-			return [];
-		}
+		//* Any config check before processing Tombstones?
+
+		this.tombstones = Array.from(tombstoneReadyNodes);
```

Comment:
Do we want to have any config option to turn this off. For example, turning both sweep and tombstone off because of some bug or something.

Reply:
Yeah adding back the tombstoneMode check.  I'm planning to remove it in the next PR (so to answer your question, I don't think we need to disable it) but for consistency here it should remain.